.github/workflows/image-builder.yml

```diff
@@ -176,14 +179,18 @@ jobs:
       # Retag these loaded images as latest, so that they get used as the base image instead of
       # the one from DockerHub
       - name: Load and tag base images
+        id: tag-base
         if: ${{ steps.check-modified.outputs.was-modified == 'true' }}
         run: |
+          build_contexts=""
           for file in ${{ runner.temp }}/*-${{ matrix.os }}.tar; do
             docker load -i "$file" | sed -n 's/Loaded image: //p' | while read -r tagged_image; do
               image="${tagged_image%:*}"
-              docker tag "$tagged_image" "$image:latest"
+              docker tag "$tagged_image" "localhost:5000/$image:latest"
```

Comment:
There was a bug loading these base images that I only uncovered with #11654. One benefit of the stacked PRs.